src/content/en/basic-workflows/qwen-image-edit.md

```
@@ -0,0 +1,178 @@
+    │   ├── qwen_2.5_vl_7b_fp8_scaled.safetensors
+    │   ├── Qwen2.5-VL-7B-Instruct.gguf                ← Only when using gguf
+    │   └── Qwen2.5-VL-7B-Instruct-mmproj-BF16.gguf    ← Only when using gguf
+    ├── 📂vae/
+    │   └── qwen_image_vae.safetensors
+    └── 📂unet/
+        └── qwen-image-edit.gguf                       ← Only when using gguf
+```
+
+### workflow
+
+![](https://gyazo.com/79b84b74171ddd5c9cfdb57bccc69f13){gyazo=image}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)